imports/blocks/add-schedule/index.js

```diff
@@ -67,11 +82,33 @@ class CartContainer extends Component {
     this.props.clearSchedules();
   }
 
-  onClick = (e) => {
+  // XXX: should remove fixed since it's not being used?
```

Comment:
This has been abstracted into a utility method by @JakeDawkins. I'm pretty sure it is merged into master now 👍